src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -40,6 +40,7 @@ internal partial class Binder
 
             ImmutableArray<string> names = default;
             ImmutableArray<RefKind> refKinds = default;
+            ImmutableArray<bool> nullCheckedOpt = default;
```

Comment:
Consider using a `BitVector` for this.

Reply:
Seems reasonable. Will look at this in a follow-up PR.